Source/Core/Common/WindowSystemInfo.h

```diff
@@ -13,6 +13,7 @@ enum class WindowSystemType
   Wayland,
   FBDev,
   Haiku,
+  UWP
```

Comment:
You might wanna add a comma at the end; that way we don't see noise in the diff next time this enum is modified.
```suggestion
  UWP,
```